makelib/golang.mk

```diff
@@ -114,9 +114,6 @@ ifeq ($(RUNNING_IN_CI),true)
 # Output checkstyle XML rather than human readable output.
 # the timeout is increased to 10m, to accommodate CI machines with low resources.
 GO_LINT_ARGS := --timeout 10m0s --out-format=checkstyle > $(GO_LINT_OUTPUT)/checkstyle.xml
-
-# Output verbose tests that can be parsed into JUnit XML.
-GO_TEST_FLAGS += -v
```

Comment:
Thanks for removing it! It should fix `go test: v flag may be set only once` error